notebooks/service/fake_store.go

```diff
@@ -0,0 +1,114 @@
+// This file is a placeholder for an actual notebooks service implementation.
+// For now it enables user experimentation with the UI in front of the notebooks
+// backend server.
+
+package service
+
+import (
+	"context"
+	"time"
+
+	"github.com/influxdata/influxdb/v2/kit/platform"
+	"github.com/influxdata/influxdb/v2/snowflake"
+)
+
+var _ NotebookService = (*FakeStore)(nil)
+
+type FakeStore struct {
+	list map[string][]*Notebook
+}
+
+func NewFakeStore() (*FakeStore, error) {
```

Comment:
Going back and forth in my head on this one, but I'm thinking:
```suggestion
func NewNotebookStore() (*FakeStore, error) {
```
To reduce the callsite diff(s) when you later swap the impl for something real.

But I see the benefit in the current name, since nobody will ever use it and expect a production-ready impl. WDYT?

Reply:
Makes sense to me. I changed the name of the function to `NewService` since it will be called like `notebookSvc.NewService()`.